### PR TITLE
Add reconciler service cmd

### PR DESCRIPTION
**Pull request by tobiscr**
Add support to start reconciler service into CLI

### CODE REVIEW

**Review by NHingerl — DISMISSED** [review]

**Review by NHingerl — APPROVED** [review]

File: cmd/service/start/options.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+		_, err = tls.X509KeyPair(crt, key)
+		if err != nil {
+			return errors.Wrap(err,
+				fmt.Sprintf("Provided TLS certificate '%s' and key '%s' is invalid", o.SSLCrt, o.SSLCrt))
+		}
+	}
+	return nil
+}
+
+func (o *Options) SslSupport() bool {
```
Comment (suleymanakbas91) [inline cmd/service/start/options.go:46]: To make it consistent with the rest, can we rename this function to SSLSupport?

File: cmd/main.go [hunk 2]
```diff
@@ -1,15 +1,124 @@
...
+
+func initViper(o *cli.Options) func() {
+	return func() {
+		//read configuration from ENV vars
+		viper.SetEnvPrefix(envVarPrefix)
+		viper.AutomaticEnv()
+
+		//read configuration from config file
+		cfgFile := getConfigFile()
+		if !file.Exists(cfgFile) {
```
Comment (suleymanakbas91) [inline cmd/main.go:82]: Would it make sense to move this check also to getConfigFile so that the both checks for the existence of file together?

**Review by suleymanakbas91 — CHANGES_REQUESTED** [review]

File: cmd/main.go (hunk 2 above)
Comment (tobiscr) [reply cmd/main.go:82]: aaah.. right. Makes sense. Will change it

File: cmd/service/start/options.go (hunk 1 above)
Comment (tobiscr) [reply cmd/service/start/options.go:46]: right, will fix it

**Review by suleymanakbas91 — APPROVED** [review]

**Pull request merged**